Python/specialize.c

```diff
@@ -791,6 +823,24 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name)
         case GETSET_OVERRIDDEN:
             SPECIALIZATION_FAIL(LOAD_ATTR, SPEC_FAIL_OVERRIDDEN);
             goto fail;
+        case GET_CALL_GETATTRIBUTE:
+        {
+            getattrofunc getattro_slot = type->tp_getattro;
+            assert(getattro_slot == _Py_slot_tp_getattro);
```

Comment:
Avoid the unused variable warning with `assert(type->tp_getattro == _Py_slot_tp_getattro);`